crates/astria-cli/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Add `contract_is_erc20` flag on bridge collect command [#1762](https://github.com/astriaorg/astria/pull/1762)
```

Comment:
```suggestion
### Added

- Add `contract_is_erc20` flag on `bridge collect-withdrawals` command [#1762](https://github.com/astriaorg/astria/pull/1762).
```

Reply:
The changes here were rolled back. The only changes now happen in its dependency, `astria-bridge-contracts`. Should this CHANGELOG still get an entry?

Reply:
Probably not if it's a small opaque change to the CLI.